product_docs/docs/migration_toolkit/55/07_invoking_mtk/08_mtk_command_options.mdx

```diff
@@ -450,21 +450,21 @@ The right side specifies a condition that must be true for each row migrated.
 
 For example, this code migrates only those countries with a `COUNTRY_ID` value that isn't equal to `AR`: 
 
-### `COUNTRIES=COUNTRY_ID<>'AR'`
+`COUNTRIES=COUNTRY_ID<>'AR'`
 
 This constraint applies to the COUNTRIES table. 
 
 You can also specify conditions for multiple tables. However, the condition for each table must be on a new line in the property file.
 
-### Example
+#### Example
```

Comment:
This seems to be the only case where "Example" is set as a heading within the context of the property descriptions. I would suggesting making this normal unbolded text as is the case for some other properties. For example, instead of
> #### Example

change it to
> Example: